### ISSUE TITLE
Lint failures not reporting to console with TSLint 5.0.0

### ISSUE THREAD

**Issue by pe8ter** [panuhorsmalahti/gulp-tslint#114]
You probably already know this, but the `tslint` method `getResult()` returns an object that no longer has the property `failureCount `. It's called `errorCount` now.

I'm filing the ticket for visibility because I noticed that the change [isn't on the new branch](https://github.com/panuhorsmalahti/gulp-tslint/blob/d76dfe1a3a73152d5af7b30f2086969c308cfa05/index.js#L138).

This change makes reporting errors and exiting with non-zero status impossible with this config:

```TypeScript
gulp.src('**/*.ts')
    .pipe(tslint({
        formatter: 'verbose'
    }))
    .pipe(tslint.report({
        emitError: true
    }))
```

gulp-tslint version: `7.0.1`
tslint version: `5.0.0`
Operating system: `macOS 10.11.6`
Node.js version: `7.7.3`

**Comment by panuhorsmalahti** [comment]
There's a PR to fix this: https://github.com/panuhorsmalahti/gulp-tslint/pull/113

It has a small thing to fix. In any case I'll get this merged in this evening (so in 10-12h probably).

**Comment by pe8ter** [comment]
Gah! I always check the Issues tab, but forget Pull Requests.

Thanks!

**Comment by amirburbea** [comment]
Now that we've merged the PR, when will the module release on npm?

**Comment by panuhorsmalahti** [comment]
Fixed in 8.0.0.
